processing/src/main/java/io/druid/query/filter/DruidFloatPredicate.java

```diff
@@ -22,9 +22,32 @@
 /**
  * FloatPredicate is only supported in Java 8+, so use this to avoid boxing when a float predicate is needed.
  */
+// All implementations are currently lambda expressions and intellij inspections wrongly complains about unused variable.
```

Comment:
Add a link to https://youtrack.jetbrains.com/issue/IDEA-191743 (when this ticket is fixed, annotation and comment could be removed)

Reply:
done. removed stale comment and added note.